### PR TITLE
Add recent articles

### PR DESCRIPTION
**Pull request by clloyd**
This is the separate page for the recent articles, with near identical functionality to the current page.

screen shot 2015-08-05 at 16 52 05

note: you'll need to ```npm i``` for this to function

### CODE REVIEW

**Comment by clloyd** [comment]
Just noticed an issue with video type article images

**Comment by clloyd** [comment]
Fixed issue

**Comment by lindseydew** [comment]
Looks good, @chrisfinch are you happy with the UI changes?

File: modules/contentApi.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                },
+                qs: {
+                    'show-fields': 'all',
+                    'show-elements': 'all',
+                    'show-tags': 'all',
+                    'to-date': moment().format('YYYY-MM-DD'),
+                    'page-size': 50
+                }
+            }, function(error, response, body) {
+                if (response.statusCode !== 200 || error) {
```
Comment (chrisfinch) [inline modules/contentApi.js]: Maybe add some caching on this response?

**Comment by chrisfinch** [comment]
Sorry for the delay! Yes, looks good :) :+1:

File: modules/contentApi.js (hunk 1 above)
Comment (lindseydew) [inline modules/contentApi.js]: Good idea about the cache - added this in the next commit

**Comment by chrisfinch** [comment]
:+1:

**Pull request merged**